scripts/change_url

```diff
@@ -103,17 +104,19 @@ then
 	mv $nginx_conf_path /etc/nginx/conf.d/$new_domain.d/$app.conf
 	# Store file checksum for the new config file location
 	ynh_store_file_checksum --file="/etc/nginx/conf.d/$new_domain.d/$app.conf"
-	domain="$new_domain"
-	path_url="$new_path"
+
 fi
 
 #=================================================
 # SPECIFIC MODIFICATIONS
 #=================================================
-# Reedit configuration files
+# UPDATE A CONFIG FILE
 #=================================================
-ynh_script_progression --message="Updating app's configuration..." --weight=1
+ynh_script_progression --message="Updating a configuration file..." --weight=1
```

Comment:
Why this change ? I mean for the user "a" is a bit weird, while "the" or "app's" might be clearer.

Reply:
because it's written like that in the other files